src/main.cpp

```diff
@@ -166,6 +192,36 @@ void loop() {
 #else
   bleServer->loop();
 #endif
+
```

Comment:
Please be aware that there is already code handling the OTA update. I don't know what happens when callint ArduinoOTA.handle() twice.